pkg/routing/localrouter.go

```diff
@@ -119,8 +119,8 @@ func (r *LocalRouter) WriteParticipantRTC(_ context.Context, roomName livekit.Ro
 	return r.writeRTCMessage(r.rtcMessageChan, msg)
 }
 
-func (r *LocalRouter) WriteRoomRTC(ctx context.Context, roomName livekit.RoomName, identity livekit.ParticipantIdentity, msg *livekit.RTCNodeMessage) error {
-	msg.ParticipantKey = participantKey(roomName, identity)
+func (r *LocalRouter) WriteRoomRTC(ctx context.Context, roomName livekit.RoomName, msg *livekit.RTCNodeMessage) error {
+	msg.ParticipantKey = participantKey(roomName, "")
```

Comment:
Is this necessary because of cloud? Doing a participant key with an empty identity could be confusing. Is it possible to make this `roomKey` or something. Sorry, I am asking without understanding the full path, but just feels a bit odd to have a participant key with an empty string.

Reply:
It's necessary because `parseParticipantKey` requires 2 parts. Changing `parseParticipantKey` and using `roomName` would also work, I just left it because the field is called `ParticipantKey`

Reply:
Got it @frostbyte73 . Yeah, my concern is that it really feels like a room key and we are using participant key in a special mode. Maybe something to clean up later.